exe/deploy-complexity.rb

```diff
@@ -27,6 +27,10 @@ def deploy(base, to, options)
     last_n_deploys = e.to_i
     last_n_deploys = nil if last_n_deploys.zero?
   end
+  opts.on("-p", "--subdir PATTERN", String,
```

Comment:
🙋  I wouldn't guess that `-p` is the same as `--subdir`, what's the connection?

Reply:
I called it `pattern` before, renamed that to `subdir` but forgot the single-letter version :). Thank you for flagging, I'll pick something more sensible!